airship.properties

```diff
@@ -1,12 +1,12 @@
 # Airship native SDK versions
-iosVersion = 17.10.0
-androidVersion = 17.7.4
+iosVersion = 17.10.1
+androidVersion = 17.8.1
 
 # Airship.Net cross-platform version
-crossPlatformVersion = 19.3.0
+crossPlatformVersion = 19.4.0
 
 # Filename of the iOS SDK zip file
-iosFrameworkZip = Airship.zip
+iosFrameworkZip = Airship-Xcode15.zip
```

Comment:
why are we reverting this? https://github.com/urbanairship/airship-dotnet/pull/13/files